rocker-compiler/src/main/java/com/fizzed/rocker/compiler/TemplateParser.java

```diff
@@ -971,6 +1020,27 @@ private void verifySwitchBlock() {
             }
             units.removeAll(toRemove);
         }
+        private void verifySwitchExpressionBlock() {
```

Comment:
This entire method may not be needed if you remove the try/catch & break support for switch expressions.

Reply:
it is needed otherwise we will generate code for spaces in between the cases which will generate non compiling java code

Reply:
Gotcha